docs/deployment-alternatives.rst

```diff
@@ -80,7 +80,7 @@ And under the containing ``server`` block, make sure to add the following rule:
 
 Ansible
 -------
-:doc:`installation` provides a complete installation workflow, which depends primarily on the
+The Ansible deployment provides a complete installation workflow which depends primarily on the
```

Comment:
Is it possible to use another type of intra-doc link here? I'd like to funnel people to the installation page as much as possible.

Reply:
@brianhelba Before it had an erroneous link to mongo's documentation.  I think wrong links are worse than no links.  I asked where this should have pointed to on slack, but no one responded. It isn't clear to me where it was supposed to reference.

Reply:
Ah. This should reference https://github.com/girder/girder/blob/master/docs/deployment.rst